NEWS.md

```diff
@@ -1,6 +1,19 @@
 # NEWS for Lrama
 
-## Lrama 0.7.0 (2025-XX-XX)
+## Lrama 0.7.0 (2025-01-18)
```

Comment:
Update the date.

Reply:
Since it is already evening, I set the date for tomorrow.